app/MMB/src/components/exploreCity/ExploreButton.js

```diff
@@ -0,0 +1,43 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, TextButton } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+type Props = {|
+  +cityName: string,
+|};
+
+const onPress = () => {
+  console.warn('TODO');
+};
+
+export default function ExploreButton(props: Props) {
+  return (
+    <View style={styles.row}>
+      <View>
```

Comment:
Did you mean to have styles applied here?

Reply:
no